artifactory/commands/transferfiles/filesdiff.go

```diff
@@ -162,13 +163,26 @@ func (f *filesDiffPhase) handleTimeFrameFilesDiff(pcWrapper *producerConsumerWra
 }
 
 func convertResultsToFileRepresentation(results []servicesUtils.ResultItem) (files []api.FileRepresentation) {
+	var pathInRepo string
```

Comment:
Looks like the definition of this variable can be moved to be right after `if result.Type == "folder" {`.